atomic_reactor/plugins/pre_check_user_settings.py

```diff
@@ -52,10 +57,35 @@ def dockerfile_checks(self):
                 "without user Dockerfile")
             return
 
+        self.label_version_check()
         self.appregistry_bundle_label_mutually_exclusive()
         self.operator_bundle_from_scratch()
         self.multistage_docker_api_check()
 
+    def label_version_check(self):
+        """Check that Dockerfile version has correct name. It means its value
+           can only contain alphanumeric characters, underscores, and dots between keywords.
+
+           Skip the check if there is no defined version
+        """
+        msg = (
+            "Dockerfile label version can only contain "
+            "alphanumeric characters, underscores, and dots between keywords"
+        )
+
+        parser = df_parser(self.workflow.builder.df_path, workflow=self.workflow)
+        dockerfile_labels = parser.labels
+        labels = Labels(parser.labels)
+        component_label = labels.get_name(Labels.LABEL_TYPE_VERSION)
+
+        if component_label not in dockerfile_labels:
```

Comment:
version is required label, and build without it would fail already in osbs-client

Reply:
Fixed